prime-router/src/test/kotlin/azure/ValidateFunctionTests.kt

```diff
@@ -192,6 +206,155 @@ class ValidateFunctionTests {
         verify(exactly = 0) { validateFunc.processRequest(any(), any()) }
     }
 
+    @Test
+    fun `test validate endpoint only client`() {
+        val metadata = UnitTestUtils.simpleMetadata
```

Comment:
I definitely realize this is continuing some redundancy/repeat logic between test cases, so I'm happy to clean this up if it's a blocker.  Just let me know!